### ISSUE TITLE
Add an Accessibility statement to the Salt site

### ISSUE THREAD

**Issue by jake-costa** [jpmorganchase/salt-ds#4663]
### Related page(s)

https://www.saltdesignsystem.com/

### Kind of issue

Missing information

### Description

We should an accessibility statement to the site to clarify our stance, share external resources and reinforce that WCAG compliance isn’t just about individual components and requires attention to the entire user experience, including component combinations, page structure, content organization, etc.

### Browser

- [ ] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Microsoft Edge

### Operating system

- [ ] macOS
- [ ] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

### Are you a JPMorgan Chase & Co. employee?

- [x] I am an employee of JPMorgan Chase & Co.

**Comment by jake-costa** [comment]
Added to internal Salt site. Will create another issue to add it to the public site as well if needed.
